lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Types.scala

```diff
@@ -12,12 +12,13 @@ object Types {
   val assetType = CASETYPEREF(
     "Asset",
     List(
-      "totalAmount" -> LONG,
-      "decimals"    -> LONG,
-      "issuer"      -> BYTESTR,
-      "reissuable"  -> BOOLEAN,
-      "scripted"    -> BOOLEAN,
-      "sponsored"   -> BOOLEAN
+      "totalAmount"     -> LONG,
+      "decimals"        -> LONG,
+      "issuer"          -> BYTESTR,
```

Comment:
issuer should be Address, not BYTESTR(as everywhere else)

Reply:
+